sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -192,12 +197,29 @@ void gnrc_rpl_send_DIS(gnrc_rpl_dodag_t *dodag, ipv6_addr_t *destination)
     _gnrc_rpl_send(pkt, NULL, destination, (dodag ? &dodag->dodag_id : NULL));
 }
 
+static bool _gnrc_rpl_check_DIS_validity(gnrc_rpl_dis_t *dis, uint16_t len)
+{
+    uint16_t expected_len = sizeof(*dis) + sizeof(icmpv6_hdr_t);
```

Comment:
@gebart I tested `static const uint16_t` at this position for the `samr21` and the iotlab-m3. In both cases the text section records 8 more bytes.